app/scripts/controllers/metametrics.test.js

```diff
@@ -604,6 +605,69 @@ describe('MetaMetricsController', function () {
     });
   });
 
+  describe('trackEvent', function () {
+    it('should call only metricsController._trackEvent if actionId is not provided', function () {
+      const metaMetricsController = getMetaMetricsController({
+        participateInMetaMetrics: false,
+      });
+      const spyTrackEvent = sinon.spy();
+      metaMetricsController._trackEvent = spyTrackEvent;
+
+      const spyCreateEvent = sinon.spy();
+      metaMetricsController.createEventFragment = spyCreateEvent;
+
+      metaMetricsController.trackEvent({}, {});
+
+      assert(spyTrackEvent.calledOnce);
+      assert(spyCreateEvent.notCalled);
+    });
+
+    it('should call createEventFragment, finalizeEventFragment if actionId is provided', function () {
```

Comment:
What does this achieve?

Reply:
Doesn't it demonstrate that the conditional being introduced to `trackEvent` - where passing an `actionId`  leads to finalizing a fragment - is working as expected?

Reply:
I tried to make test cases more clear in the description, plz check.